packaging/rhel/syslog-ng.spec

```diff
@@ -314,7 +314,7 @@ export GEOIP_LIBS=-lGeoIP
     --with-module-dir=%{_libdir}/%{name} \
     --with-systemdsystemunitdir=%{_unitdir} \
     --with-ivykis=system \
-%if 0%{?rhel}
+%if 0%{?rhel} == 7
```

Comment:
Thanks for contributing this back.

Shouldn't we keep using TCP wrappers in case of RHEL6 too (`0%{?rhel} <= 7`)?

Reply:
I have some strong doubts, that this spec file would work on RHEL 6, as it depends on systemd.

Reply:
The Centos 6 support has been added back in PR #2860. AFAIK it uses this spec file.

Reply:
Yepp:

%if 0%{?rhel} >= 7 || 0%{?fedora} >= 28
%bcond_without systemd

Reply:
Well, you might be able to compile syslog-ng using this spec file (did not test it), but not install or run it. RHEL 6 is using sysvinit, but there are no init scripts included. And even if it does not use systemd to compile syslog-ng, the pre/post install scripts depend on systemd. This is why I have RHEL 6 support in a separate repository with a separate spec file...

Reply:
OK, not that it has any practical use, but I edited it using the web interface to <= from ==